table.csv

```diff
@@ -62,7 +62,8 @@ keccak-224,keccak has variable output length. The number specifies the core leng
 keccak-256,         ,                         0x1B
 keccak-384,         ,                         0x1C
 keccak-512,         ,                         0x1D
-murmur3,            ,                         0x22
+murmur3-128,        ,                         0x22
+murmur3-32,         ,                         0x23
```

Comment:
Let's be careful about allocating another single-byte code. Really, we shouldn't have allocated single byte codes to most of these but, IIRC, we did that *before* we combined the various codec tables.

Reply:
I took it from the JS Multihash implementation https://github.com/multiformats/js-multihash/blob/443a8ebdd145582f2cac6e708ffe890401d5de08/src/constants.js#L22

Reply:
Damn.

Reply:
It's even in the multihash RFC draft: https://tools.ietf.org/html/draft-multiformats-multihash-00